src/core/analogsensor.cpp

```diff
@@ -157,12 +157,13 @@ void AnalogSensor::reload(bool get_nvs) {
             if (!found) {
                 // check if the GPIO is valid before registering. If not, force set the sensor to disabled, but don't remove it
                 // should only trigger if uploading a customization file with invalid gpios.
+                AnalogType type = static_cast<AnalogType>(sensor.type);
                 if (!EMSESP::system_.is_valid_gpio(sensor.gpio)) {
```

Comment:
This check is obsolete, you now check in WebcusomizationService and remove the sensor.